From: Jan Engelhardt <jengelh@medozas.de>

build: fix compiler warnings about prototypes

p_enemy.c: In function 'A_KeenDie':
p_enemy.c:1023:3: warning: call to function 'A_Fall' without a real prototype
p_enemy.h:59:6: note: 'A_Fall' was declared here
p_enemy.c: In function 'A_Chase':
p_enemy.c:1147:5: warning: call to function 'A_FaceTarget' without a real prototype
p_enemy.h:63:6: note: 'A_FaceTarget' was declared here
p_enemy.c: In function 'A_StartFire':
p_enemy.c:1755:3: warning: call to function 'A_Fire' without a real prototype
p_enemy.h:72:6: note: 'A_Fire' was declared here
p_enemy.c: In function 'A_FireCrackle':
p_enemy.c:1761:3: warning: call to function 'A_Fire' without a real prototype
p_enemy.h:72:6: note: 'A_Fire' was declared here
p_enemy.c: In function 'A_PainDie':
p_enemy.c:2067:3: warning: call to function 'A_Fall' without a real prototype
p_enemy.h:59:6: note: 'A_Fall' was declared here
p_enemy.c: In function 'A_CloseShotgun2':
p_enemy.c:2326:3: warning: call to function 'A_ReFire' without a real prototype
p_pspr.h:100:6: note: 'A_ReFire' was declared here
p_enemy.c: In function 'A_SpawnSound':
p_enemy.c:2457:3: warning: call to function 'A_SpawnFly' without a real prototype
p_enemy.h:105:6: note: 'A_SpawnFly' was declared here

---
 src/p_enemy.h |  122 +++++++++++++++++++++++++++++-----------------------------
 1 file changed, 61 insertions(+), 61 deletions(-)

Index: prboom-plus-2.5.1.1/src/p_enemy.h
===================================================================
--- prboom-plus-2.5.1.1.orig/src/p_enemy.h
+++ prboom-plus-2.5.1.1/src/p_enemy.h
@@ -53,66 +53,66 @@ extern struct brain_s {         /* killo
 // allows more extensive changes (see d_deh.c)
 
 // Doesn't work with g++, needs actionf_p1
-void A_Explode();
-void A_Pain();
-void A_PlayerScream();
-void A_Fall();
-void A_XScream();
-void A_Look();
-void A_Chase();
-void A_FaceTarget();
-void A_PosAttack();
-void A_Scream();
-void A_SPosAttack();
-void A_VileChase();
-void A_VileStart();
-void A_VileTarget();
-void A_VileAttack();
-void A_StartFire();
-void A_Fire();
-void A_FireCrackle();
-void A_Tracer();
-void A_SkelWhoosh();
-void A_SkelFist();
-void A_SkelMissile();
-void A_FatRaise();
-void A_FatAttack1();
-void A_FatAttack2();
-void A_FatAttack3();
-void A_BossDeath();
-void A_CPosAttack();
-void A_CPosRefire();
-void A_TroopAttack();
-void A_SargAttack();
-void A_HeadAttack();
-void A_BruisAttack();
-void A_SkullAttack();
-void A_Metal();
-void A_SpidRefire();
-void A_BabyMetal();
-void A_BspiAttack();
-void A_Hoof();
-void A_CyberAttack();
-void A_PainAttack();
-void A_PainDie();
-void A_KeenDie();
-void A_BrainPain();
-void A_BrainScream();
-void A_BrainDie();
-void A_BrainAwake();
-void A_BrainSpit();
-void A_SpawnSound();
-void A_SpawnFly();
-void A_BrainExplode();
-void A_Die();
-void A_Detonate();        /* killough 8/9/98: detonate a bomb or other device */
-void A_Mushroom();        /* killough 10/98: mushroom effect */
-void A_Spawn();           // killough 11/98
-void A_Turn();            // killough 11/98
-void A_Face();            // killough 11/98
-void A_Scratch();         // killough 11/98
-void A_PlaySound();       // killough 11/98
-void A_RandomJump();      // killough 11/98
-void A_LineEffect();      // killough 11/98
+void A_Explode(mobj_t *);
+void A_Pain(mobj_t *);
+void A_PlayerScream(mobj_t *);
+void A_Fall(mobj_t *);
+void A_XScream(mobj_t *);
+void A_Look(mobj_t *);
+void A_Chase(mobj_t *);
+void A_FaceTarget(mobj_t *);
+void A_PosAttack(mobj_t *);
+void A_Scream(mobj_t *);
+void A_SPosAttack(mobj_t *);
+void A_VileChase(mobj_t *);
+void A_VileStart(mobj_t *);
+void A_VileTarget(mobj_t *);
+void A_VileAttack(mobj_t *);
+void A_StartFire(mobj_t *);
+void A_Fire(mobj_t *);
+void A_FireCrackle(mobj_t *);
+void A_Tracer(mobj_t *);
+void A_SkelWhoosh(mobj_t *);
+void A_SkelFist(mobj_t *);
+void A_SkelMissile(mobj_t *);
+void A_FatRaise(mobj_t *);
+void A_FatAttack1(mobj_t *);
+void A_FatAttack2(mobj_t *);
+void A_FatAttack3(mobj_t *);
+void A_BossDeath(mobj_t *);
+void A_CPosAttack(mobj_t *);
+void A_CPosRefire(mobj_t *);
+void A_TroopAttack(mobj_t *);
+void A_SargAttack(mobj_t *);
+void A_HeadAttack(mobj_t *);
+void A_BruisAttack(mobj_t *);
+void A_SkullAttack(mobj_t *);
+void A_Metal(mobj_t *);
+void A_SpidRefire(mobj_t *);
+void A_BabyMetal(mobj_t *);
+void A_BspiAttack(mobj_t *);
+void A_Hoof(mobj_t *);
+void A_CyberAttack(mobj_t *);
+void A_PainAttack(mobj_t *);
+void A_PainDie(mobj_t *);
+void A_KeenDie(mobj_t *);
+void A_BrainPain(mobj_t *);
+void A_BrainScream(mobj_t *);
+void A_BrainDie(mobj_t *);
+void A_BrainAwake(mobj_t *);
+void A_BrainSpit(mobj_t *);
+void A_SpawnSound(mobj_t *);
+void A_SpawnFly(mobj_t *);
+void A_BrainExplode(mobj_t *);
+void A_Die(mobj_t *);
+void A_Detonate(mobj_t *);        /* killough 8/9/98: detonate a bomb or other device */
+void A_Mushroom(mobj_t *);        /* killough 10/98: mushroom effect */
+void A_Spawn(mobj_t *);           // killough 11/98
+void A_Turn(mobj_t *);            // killough 11/98
+void A_Face(mobj_t *);            // killough 11/98
+void A_Scratch(mobj_t *);         // killough 11/98
+void A_PlaySound(mobj_t *);       // killough 11/98
+void A_RandomJump(mobj_t *);      // killough 11/98
+void A_LineEffect(mobj_t *);      // killough 11/98
 
 #endif // __P_ENEMY__
